src/core/dbus.c

```diff
@@ -56,13 +56,12 @@ int bus_send_pending_reload_message(Manager *m) {
         if (!m->pending_reload_message)
                 return 0;
 
-        /* If we cannot get rid of this message we won't dispatch any
-         * D-Bus messages, so that we won't end up wanting to queue
-         * another message. */
+        /* If we cannot get rid of this message we won't dispatch any D-Bus messages, so that we won't end up wanting
+         * to queue another message. */
```

Comment:
Nitpick: since you (or your $EDITOR) splitted the comments into 2 lines, why not making those 2 lines the same length ? That would avoid having a line overly long which is useless in this case.

(yes my monitor is not as wide as yours ;) )

Reply:
see CODING_STYLE regarding line length